content/docs/csidriver/_index.md

```diff
@@ -16,7 +16,7 @@ The CSI Drivers by Dell implement an interface between [CSI](https://kubernetes-
 {{<table "table table-striped table-bordered table-sm">}}
 | Features                 | PowerMax | PowerFlex | Unity XT  | PowerScale | PowerStore |
 |--------------------------|:--------:|:---------:|:---------:|:----------:|:----------:|
-| CSI Driver version       | 2.11.0   | 2.12.0    | 2.12.0    | 2.11.0     | 2.11.0     |
```

Comment:
all driver versions should be 2.12.0

Reply:
@rishabhatdell 
Changes in patch release-1.11.1 branch should have been updated in main when patch was releases.
But seems it was missed, that's why we have raised PR to update main branch with release-1.11.1 branch.
I think there would be another PR to update pending driver versions.